### DISCUSSION TITLE
Layout Analysis:Can we fine-tune Chinese model with CDLA dataset and my own native languages dataset together?

### DISCUSSION THREAD

**Discussion by TapendraBaduwal** [PaddlePaddle/PaddleOCR#8101] [General]
The layout analysis Chinese model trained on the CDLA dataset, did not work in some  case of my native languages.
Can we fine-tune Chinese model with CDLA dataset and my own native languages dataset together?

**Comment by an1018** [comment]
If the amout of your dataset is sufficient, you can retrain the model on your own data, please refer to [Layout analysis document](https://github.com/PaddlePaddle/PaddleOCR/blob/dygraph/ppstructure/layout/README.md)
Also you can fine-tune with [the pretrained model](https://github.com/PaddlePaddle/PaddleOCR/blob/dygraph/ppstructure/docs/models_list_en.md#1-layout-analysis) of CDLA or PubLayNet

**Reply by masoudMZB** [reply]
thanks for response. can you say how much data is sufficient data? I need number to create dataset. thanks
